certified-connectors/OpenExperience/apiDefinition.swagger.json

```diff
@@ -0,0 +1,249 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Open Experience",
+    "description": "This is an Open Experience connector, which enables you to integrate the Open Experience services with your flows",
+    "version": "1.0",
+    "contact": {
+      "name": "Open Experience Support",
+      "url":"https://openexperience.de/de/hilfe/MS_Datenschnittstelle/MS_Power_Automation_Connector",
+      "email":"support@openexperience.de"
+    }
+  },
+  "host": "openexperience.de",
+  "basePath": "/teamssync",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/connector/createProject": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@oxTomaszJ Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger.